packages/internet-header/cypress/e2e/search.cy.ts

```diff
@@ -4,7 +4,8 @@ import mockStaoCacheTypes from '../fixtures/internet-header/staocache-types.json
 import { copyConfig, prepare } from '../support/prepare-story';
 
 describe('search', () => {
-  const searchButton = '#post-internet-header-search-button';
+  const searchButton = '#post-internet-header-search-button[aria-expanded=false]';
+  const closeButton = '#post-internet-header-search-button[aria-expanded=true]';
```

Comment:
For some reason, Cypress clicks multiple time on the same button with the same id.